jekyll/_cci2/vs-code-extension-overview.adoc

```diff
@@ -11,34 +11,176 @@ contentTags:
 :toc: macro
 :toc-title:
 
-The CircleCI VS Code extension helps you avoid context-switching between VS Code and your browser.
+The CircleCI Visual Studio Code extension enables users to manage their CircleCI pipelines directly from VS Code. 
 
 [#introduction]
 == Introduction
 
-The CircleCI VS Code extension lets you visualize and manage CircleCI pipelines directly from your IDE, as well as providing contextual help when creating, modifying and editing CircleCI YAML config files. 
+The VS Code extension includes:
 
-The extension allows you to:
+- The **Pipeline Manager**, which displays pipeline status live within the integrated development environment (IDE). The pipeline manager allows you to identify issues and take immediate action on your pipelines without context-switching between VS Code and your browser.
```

Comment:
I think it's fine and good to spell out IDE, maybe switching round so the expanded words on within the brackets would work better. It may well be rare to see the words, but it might help some readers have a full understanding, and helps us be very specific about what we are writing about (VS code being and more than just an editor I mean).